### ISSUE TITLE
what are the values of x and z in table view controller(washAction)?

### ISSUE THREAD

**Issue by Parameshvadivel** [asong4211/iNav#2]
First, Thanks for awesome ar projects. Here the values of x and z in table view controller (washAction button). 
@adbht @asong4211

**Comment by Parameshvadivel** [comment]
@IBAction func washAction(_ sender: UIButton) {
        
        self.data = [
            "x":[0.134249121,0.449385971,0.55,0.55,0.70,1.50,1.8,1.23434155,-1.68046713,-6.16991758,-12.7871971,-17.5291958,],
            "y":[-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,],
            "z":[-0.579975545,-3.65786052,-6.56569815,-10.5181684,-13.2360897,-18.7201805,-21.5751476,-24.181345,-25.7771645,-26.8030167,-27.9402733,-29.1691208]
        ]
        self.performSegue(withIdentifier: "toAR", sender: sender)
    }

The value X and Z are lat long??

**Comment by adbht** [comment]
No, they are relative to the starting point. In other words, the origin is the position where the user scans the QR code. From there, these points are used to plot the 3D sphere in helping the user to navigate to their desired destination.

**Comment by Parameshvadivel** [comment]
how to modify x and z values based on my project requirements. I dont have any QR code. Is it possible to show direction based on lat long?

How to show sphere lines based on my requirements?

**Comment by Parameshvadivel** [comment]
can you send qr code?
what information stored in qr code?

**Comment by Parameshvadivel** [comment]
> No, they are relative to the starting point. In other words, the origin is the position where the user scans the QR code. From there, these points are used to plot the 3D sphere in helping the user to navigate to their desired destination.

if QR code position is changed to different location, whether it will show paths form that position?

**Comment by Parameshvadivel** [comment]
> No, they are relative to the starting point. In other words, the origin is the position where the user scans the QR code. From there, these points are used to plot the 3D sphere in helping the user to navigate to their desired destination.

How to did you find x, y, z node position values?
Did you use any map created for indoor navigation?
